scripts/start.py

```diff
@@ -24,6 +24,7 @@
 import fileinput
 import os
 import re
+import shutil
```

Comment:
is this used anywhere, ditto here and above?

Reply:
No, it isn't used.

Reply:
Removed.